### DISCUSSION TITLE
Fundamentos do C#  - Declaração de constantes sem inicialização explícita de valor

### DISCUSSION THREAD

**Discussion by JWES1971** [balta-io/forum#529] [Tira Dúvidas] [answered]
Olá a todos!,
Estou realizando o curso 
Fundamentos do C# , e na aula onde é apresentado o conceito de constantes é apresentado ser possível definir uma constante sem inicializá-la explicitamente com um valor.
Ex.:  const int IDADE_MINIMA; // inicializada com valor Zero por padrão

Contudo estou obtendo um erro, indicando que o valor deve ser fornecido (A const field requires a value to be provided).

Estou usando .NET 8.0.103. Essa versão não mais suporta a declaração sem inicialização explícita do valor da constante?
![Captura de tela de 2024-03-26 19-36-28](https://github.com/balta-io/forum/assets/119546756/0474d95e-ae0b-41ce-9257-3e42c45e9afb)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @JWES1971, como vai?

No caso, você está recebendo um `warning`(Aviso) de que está criando uma variável (Ou constante) mas não está usando ela em nenhum lugar.

Assim que utilizá-la, em um `Console.Write(IDADE_MINIMA)` por exemplo, este `warning` irá desaparecer.
